frontend/src/pages/RequestorPage/ResponsePanel.tsx

```diff
@@ -426,7 +426,7 @@ function ResponseBody({
     }
 
     if (body?.type === "json") {
-      const prettyBody = JSON.stringify(JSON.parse(body.value), null, 2);
+      const prettyBody = JSON.stringify(safeParseJson(body.value), null, 2);
```

Comment:
shoehorned this in here, will be useful if we ever get a partial response back from a server for some reason ((happened to me, not sure why))

Reply:
the alternative here is that we crash the entire app